Add hints about loglevel (#3799)
owncloud/docs: modules/admin_manual/pages/installation/docker/index.adoc

== Troubleshooting

=== Admin Settings

When running under docker, the admin user cannot control certain settings in the WebUI, instead they are now controlled by environment variables.

> Changing these variables requires stopping and restarting the container with extra `docker -e ...` parameters, or with new entries in docker-compose's `.env` file.

Reviewer: ```suggestion
Changing these variables requires stopping and restarting the container with extra `docker -e ...` parameters or with new entries in the `.env` file for docker-compose.
```

==== Logging

> The loglevel is fixed at 2, "Warnings, errors, and fatal issues".

Reviewer: ```suggestion
The loglevel is set to the fixed value 2: "Warnings, errors, and fatal issues".
```

> ==== Logging
> The loglevel is set to the fixed value 2: "Warnings, errors, and fatal issues".

Reviewer 2: ```suggestion
==== Logging

The loglevel is set to the fixed value 2: "Warnings, errors, and fatal issues".
```

== Troubleshooting

=== Admin Settings

> When running under docker, the admin user cannot control certain settings in the WebUI, instead they are now controlled by environment variables.
> Changing these variables requires stopping and restarting the container with extra `docker -e ...` parameters or with new entries in the `.env` file for docker-compose.

Reviewer 2: ```suggestion
When running under docker, the admin user cannot control certain settings in the WebUI, instead they are now controlled by environment variables. Changing these variables requires stopping and restarting the container with extra `docker -e ...` parameters or with new entries in the `.env` file for docker-compose.
```

==== Logging

The loglevel is set to the fixed value 2: "Warnings, errors, and fatal issues".

> .To get the highest log level "Everything" (including debug output), use
> +
> [source,console]

Reviewer 2: ```suggestion
.To get the highest log level "Everything" (including debug output), use

[source,console]
```
